utils/seed_asset.py

```diff
@@ -19,18 +22,28 @@
 
 
 if __name__ == '__main__':
-    filename = input('Enter csv filename (without .csv): ').strip()
+    filename = input('Enter csv filename '
```

Comment:
Rename `filename` to be more descriptive, considering the url use case.

Reply:
Okay, will do that.